jackson/src/main/resources/reference.conf

```diff
@@ -6,7 +6,7 @@ lagom.serialization.json {
   # The Jackson JSON serializer will register these modules.
   # It is also possible to use jackson-modules = ["*"] to dynamically
   # find and register all modules in the classpath.  
-  jackson-modules = [
+  jackson-modules = ${lagom.serilization.json.jackson-modules} [
```

Comment:
Thanks, Jaie!

The build is failing with:

```
[error] Caused by: com.typesafe.config.ConfigException$UnresolvedSubstitution: reference.conf @ file:/home/travis/build/lagom/lagom/jackson/target/classes/reference.conf: 9: Could not resolve substitution to a value: ${lagom.serilization.json.jackson-modules}
```

There's a typo in there (serilization -> serialization) but that might not be the problem... if this is the first definition of the key that is loaded, then I think it will still fail.

I think you need to use the syntax `${?lagom.serialization.json.jackson-modules}`, which [silently ignores undefined keys](https://github.com/typesafehub/config#optional-system-or-env-variable-overrides).

Of course, if you had used the optional syntax and the key name still had a typo, the build would have passed but this wouldn't have worked.

Another option is to do this the way [`play.modules.enabled` is defined](https://github.com/playframework/playframework/blob/master/framework/src/play/src/main/resources/reference.conf#L815-L819), which is more verbose but safer:

```
jackson-modules += "com.fasterxml.jackson.module.paramnames.ParameterNamesModule"
jackson-modules += "com.fasterxml.jackson.datatype.jdk8.Jdk8Module"
// etc
```